diff --git a/data/org.freedesktop.Tuhi.appdata.xml.in.in b/data/org.freedesktop.Tuhi.appdata.xml.in.in
index 5ab9d11..3ef93cd 100644
--- a/data/org.freedesktop.Tuhi.appdata.xml.in.in
+++ b/data/org.freedesktop.Tuhi.appdata.xml.in.in
@@ -41,7 +41,8 @@
   <url type="homepage">https://github.com/tuhiproject/tuhi/</url>
   <url type="bugtracker">https://github.com/tuhiproject/tuhi/issues</url>
   <url type="help">https://github.com/tuhiproject/tuhi/wiki</url>
-  <project_group>GNOME</project_group>
+  <url type="vcs-browser">https://github.com/tuhiproject/tuhi/</url>
+  <developer_name>The Tuhi Project</developer_name>
 
   <translation type="gettext">tuhi</translation>
 
